src/docs/learn.md

```diff
@@ -0,0 +1,92 @@
+# Learn
+
+`learn` command allows allows training a network from training data.
+
+As all commands in stockfish `learn` can be invoked either from command line (as `stockfish.exe learn ...`, but this is not recommended because it's not possible to specify UCI options before `learn` executes) or in the interactive prompt.
+
```

Comment:
Could we describe below?
- It is recommended to set `EnableTranspositionTable` to `false`.
- It is recommended to set `PruneAtShallowDepth` to `false`.
- It is needed to set `Use NNUE` to `pure`.